src/core/DisplayGroup.cpp

```diff
@@ -163,6 +176,17 @@ void DisplayGroup::setContentWindows( ContentWindowPtrs contentWindows )
     }
 }
 
+DisplayGroup& DisplayGroup::operator=( const DisplayGroup& displayGroup )
+{
+    setContentWindows( displayGroup.getContentWindows() );
+    setShowWindowTitles( displayGroup.getShowWindowTitles() );
+    coordinates_.setX( displayGroup.getCoordinates().x() );
```

Comment:
Also, you have access to the other objects private members (it's the same class) so you don't need to use the getters.